generators/python/src/fern_python/snippet/type_declaration_snippet_generator.py

```diff
@@ -22,6 +23,8 @@ def __init__(
         self._generate_enum = enum
         self._generate_object = object
         self._generate_discriminated_union = discriminated_union
+        # TODO: implement undiscriminated_union snippets
+        # self._generate_undiscriminated_union = undiscriminated_union
```

Comment:
Thanks for adding a TODO here -- do you mind adding this TODO in linear as well so we can track